### DISCUSSION TITLE
multiple crash on ubunutu 22.04 when using menu

### DISCUSSION THREAD

**Discussion by dominix** [transmission/transmission#3808] [Support] [answered]
Transmission do not crash when just downloading or sharing, 
On a random way, when using "change location", "properties" (file rename), "properties" (change ratio), or preferences in main menu transmission just crash, not saving last changes (in file rename, relocation, or whatever.
This was clearly not the case on 20.04
dont know were to report bug, could be GTK, Transmission, Mate, or any sub-project
thanks for your feedback or directions

**Comment by dirkChalet** [comment]
I see the same

**Comment by ckerr** [comment, marked as answer]
Please file a bug report at https://github.com/transmission/transmission/issues preferably using one of the 4.0.0 betas.
